### PR TITLE
Request for merge Some feature 2 to dev

### PR DESCRIPTION
**Pull request by tanhung3101**
Request for merge Some feature 2 to dev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # learn-nodejs
 fix dev_1
-fix dev_2
```
Comment (hungnht) [inline README.md:3]: why comment that out?

Comment (hungnht) [inline README.md:3]: Test again

**Review by hungnht — COMMENTED** [review]
review again

File: README.md (hunk 1 above)
Comment (tanhung3101) [reply README.md:3]: why nott?

**Review by hungnht — APPROVED** [review]

**Pull request merged**